content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -20,45 +21,51 @@ This document will hopefully help you to figure out what's going wrong.
 ## Conventions
 
 Throughout this doc you will see various commands that you can run.  Some
-commands need to be run within a `Pod`, others on a Kubernetes `Node`, and others
+commands need to be run within a Pod, others on a Kubernetes Node, and others
 can run anywhere you have `kubectl` and credentials for the cluster.  To make it
 clear what is expected, this document will use the following conventions.
 
-If the command "COMMAND" is expected to run in a `Pod` and produce "OUTPUT":
+If the command "COMMAND" is expected to run in a Pod and produce "OUTPUT":
 
 ```shell
 u@pod$ COMMAND
```

Comment:
nit: this style guide recommends not including the prompt.

If I want to tell the reader to run this on a node, or inside a shell in a container, I usually put a comment inside the code block on the line above the command.

Might be worth fixing that too?

Reply:
ACK